src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1124,6 +1122,6 @@ external lteq   ::= collapse LTEQ LT EQ       { name = "<=" }
 external oror   ::= collapse OROR OR OR       { name = "||" }
 
 // Trailing commas are allowed
-private meta comma_separated_list ::= <<param>> ( ',' <<param>> ) * ','?
+private meta comma_separated_list ::= <<fn_param>> ( ',' <<fn_param>> ) * ','?
```

Comment:
>Grammar-Kit gets crazy when it's fed with << param >> instead of <<param>> (notice whitespaces)

I have no emoji for this.

Yes, can you change all occurrences of `<< foo >>` to `<<foo>>` to align with more common grammar kit style. I would have done it myself, but I don't want to interfere with your other PRs.

Thanks a lot for all this work!